recipe/meta.yaml

```diff
@@ -1,19 +1,16 @@
 {% set name = "gevent" %}
-{% set version = "22.10.2" %}
-{% set sha256 = "1ca01da176ee37b3527a2702f7d40dbc9ffb8cfc7be5a03bfa4f9eec45e55c46" %}
+{% set version = "23.9.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 72c002235390d46f94938a96920d8856d4ffd9ddf62a303a0d7c118894097e34
   patches:   # [unix]
     - 0001-Allow-include-flags-to-be-present-in-CFLAGS.patch        # [unix]
     - 0002-Allow-library-flags-to-be-present-in-LDFLAGS.patch       # [unix]
-    - 0003-Do-not-use-hard-coded-path-for-file-command.patch        # [unix]
```

Comment:
Could you re-sync them with conda-forge please?

Reply:
Thanks for pointing me out. I refreshed patches